gradle-baseline-java/src/main/groovy/com/palantir/baseline/extensions/BaselineErrorProneExtension.java

```diff
@@ -71,6 +71,7 @@
             // TODO(ckozak): re-enable pending scala check
             // "ThrowSpecificity",
             "UnsafeGaugeRegistration",
+            "PreferImmutableStreamExCollections",
```

Comment:
If someone out there is relying on the _mutability_ of their streamex collections, e.g.:

```java
Map<String, String> foo = myStream.toMap();
foo.remove("bar");
return foo;
```

Then this auto-fix will make a change which still _compiles_ but would throw when they run this.  This _should_ get caught by tests, but if they also don't have tests then this could break someone in production.